skyrim-platform/src/platform_se/skyrim_platform/FridaHooks.cpp

```diff
@@ -42,7 +42,8 @@ enum _ExampleHookId
   APPLY_MASKS_TO_RENDER_TARGET,
   RENDER_MAIN_MENU,
   SEND_EVENT,
-  SEND_EVENT_ALL
+  SEND_EVENT_ALL,
+  CONSOLE_VPRINT
```

Comment:
@Pospelove BTW I think it's better to add trailing comma in such cases. Modern C++ should allow that, if I remember correctly

Reply:
@nic11 ok